kivy/graphics/gl_debug_logger.pxi

```diff
@@ -0,0 +1,10 @@
+from kivy.graphics.c_opengl cimport GLenum, glGetError
+from kivy.logger import Logger
+include "config.pxi"
+
+cdef inline void log_gl_error(str note):
+    IF not HIDE_GL_LOG:
```

Comment:
Does this code works with HIDE_GL_LOG? If the IF is evaluate to False, the result will be an empty function, not even an pass.